runtime/doc/options.txt

```diff
@@ -6104,11 +6104,27 @@ A jump table for the options with a short description can be found at |Q_op|.
 	( -   Start of item group.  Can be used for setting the width and
 	      alignment of a section.  Must be followed by %) somewhere.
 	) -   End of item group.  No width fields allowed.
-	T N   For 'tabline': start of tab page N label.  Use %T after the last
-	      label.  This information is used for mouse clicks.
-	X N   For 'tabline': start of close tab N label.  Use %X after the
-	      label, e.g.: %3Xclose%X.  Use %999X for a "close current tab"
-	      mark.  This information is used for mouse clicks.
+	T N   For 'tabline': start of tab page N label.  Use %T or %X to end 
+	      the label.  This information is used for mouse clicks.
```

Comment:
This information → The information (or This piece of information)

"This information" is grammatically incorrect, as "information" is always plural in English.